# Metagenomics of Cheese Rind Microbiomes :cheese: :dna:

This repository contains code, raw data, and results files used for analyzing metagenomes from five distinct washed rind cheese microbiomes. This repository is associated with the figures produced for the [Paired long- and short-read metagenomics of cheese rind microbial communities at multiple time points](https://doi.org/10.57844/arcadia-0zvp-xz86) pub. Raw reads and assemblies can be found in the [European Nucleotide Archive under Project PRJEB58160](https://www.ebi.ac.uk/ena/browser/view/PRJEB58160). This includes a total of 24 metagenomic assemblies from five distinct washed rind cheese microbiomes. For three cheeses, paired short and long read sequencing data from three different timepoints in the cheese aging process is available.

The "cheesegenomes-k31-scaled1k" within 'databases' contains sourmash signatures from previously sequenced microbial genomes from cheese. These signatures were used in the 'sourmash gather' analysis. Sourmash signatures used to create the MDS plot of the assemblies can be found in the "sourmashsigs" folder within 'processed_data'.

## Test data for Arcadia-Science/metagenomics workflow
This repository contains documentation and code for analyzing short- and long-read metagenomes from the cheeses described above as part of validating the [Arcadia-Science/metagenomics](https://github.com/Arcadia-Science/metagenomics) Nextflow workflow. Specifically the R code in `scripts/TMI-sourmash-explore.R` was used to exemplify visualizations that can be output from the output files of `sourmash` using the R package `sourmashconsumr`. The `sourmash` output files used for this analysis are available on [Zenodo](https://zenodo.org/record/7968234) as they are different from what is in the `processed_data` directory.
